fintech-examples/fintech-ui/src/app/login/login.component.ts

```diff
@@ -16,7 +16,10 @@ export class LoginComponent implements OnInit {
   ngOnInit() {
     this.loginForm = this.formBuilder.group({
       username: ['', [Validators.required, Validators.pattern('^[a-z0-9]+$')]],
-      password: ['', [Validators.required, Validators.pattern('.{4}')]]
+      password: ['', Validators.required]
```

Comment:
Since backend requires 4 character password to register new user - this one was used